rhcephcompose/gather/chacra.py

```diff
@@ -0,0 +1,25 @@
+from rhcephcompose import Build
+from rhcephcompose.log import log
+
+
+def query(builds_file, chacra_url, whitelist, ssl_verify):
+    """
+    Find all the artifacts we will need from a given build list text file.
+
+    :param builds_file: "builds" text file for a distro.
+    :param chacra_url: chacra server to query
+    :param whitelist: whitelist of deb package names to save
+    :param ssl_verify: verify HTTPS connection or not
+    :returns: list of rhcephcompose Build objects
+    """
+    build_ids = [line.rstrip('\n') for line in open(builds_file)]
```

Comment:
this call to `open()` would ideally need to be closed

Reply:
great catch, I'll do this in a separate PR

Reply:
I've pushed https://github.com/red-hat-storage/rhcephcompose/pull/83 to do this